src/main/resources/application.properties

```diff
@@ -0,0 +1,12 @@
+management.endpoints.enabled-by-default=false
+management.endpoints.web.base-path=/account-validator
+management.endpoints.web.path-mapping.health=/healthcheck
+management.endpoint.health.show-details=never
+management.endpoint.health.enabled=true
+
+chs.kafka.api.endpoint=${CHS_KAFKA_API_URL:localhost}
```

Comment:
Does this service use kafka? If not why is this property needed?

Reply:
I don't think we need 
Removed